pkg/controller/common/cni/cni_test.go

```diff
@@ -0,0 +1,26 @@
+package cni
+
+import (
+	"testing"
+
+	"github.com/maistra/istio-operator/pkg/controller/common"
+	"github.com/maistra/istio-operator/pkg/controller/common/test/assert"
+	"sigs.k8s.io/controller-runtime/pkg/manager"
+)
+
+func TestInitConfig_disablingCNI(t *testing.T) {
```

Comment:
Can you add a test that ensures this is enabled by default?  I'm concerned that if this is not specified in the config that it will default to false.  @luksa, would you be okay if this was renamed `CNIDisabled`?

Reply:
I'd prefer to keep `CNIEnabled` to prevent double negatives, but I see your point. Is there a way we can make it default to `true`, like we can do with cmd-line arguments?

Reply:
@luksa @rcernich 
Currently, I have defaulted it to `true` in an `init()` function, same way as for the `webhookConfiguration` switch.
https://github.com/maistra/istio-operator/pull/869/files#diff-2e3990f5984ac30e31d8e7167db0157324bd18330d8502da656e93cc620355fdL14-L16